src/main/java/com/minecolonies/coremod/tileentities/TileEntityBakerBuilding.java

```diff
@@ -0,0 +1,78 @@
+package com.minecolonies.coremod.tileentities;
+
+import com.minecolonies.coremod.colony.buildings.AbstractBuilding;
+import com.minecolonies.coremod.colony.buildings.BuildingBaker;
+import com.minecolonies.coremod.entity.ai.citizen.baker.BakerEnums;
+import com.minecolonies.coremod.entity.ai.citizen.baker.BakingProduct;
+import net.minecraft.block.BlockFurnace;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.init.Blocks;
+import net.minecraft.item.crafting.RecipesFood;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.item.crafting.RecipesFood'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)